apps/api-extractor/src/analyzer/Span.ts

```diff
@@ -404,39 +431,69 @@ export class Span {
   }
 
   /**
-   * Recursive implementation of `getModifiedText()` and `writeModifiedText()`.
+   * Returns a diagnostic dump of the tree, showing the SpanModification settings for each nodde.
    */
-  private _writeModifiedText(options: IWriteModifiedTextOptions): void {
-    if (this.modification.indentDocComment) {
-      if (options.indentDocCommentState !== IndentDocCommentState.Inactive) {
-        throw new InternalError('indentDocComment cannot be nested');
+  public getModifiedDump(indent: string = ''): string {
```

Comment:
(`getDump()` and `getModifiedDump()` are not used by the production code; they are manually invoked in the debugger to help you inspect the data structure.)